charts/navi-attractor/values.yaml

```
@@ -0,0 +1,356 @@
+
+# @param pdb.enabled If PDB is enabled for the service.
+# @param pdb.minAvailable How many pods must be available after the eviction.
+# @param pdb.maxUnavailable How many pods can be unavailable after the eviction.
+
+pdb:
+  enabled: false
+  minAvailable: ''
+  maxUnavailable: 1
+
+
+# @section Navi-Attractor application settings
+
+# @param attractor.appPort Container port for HTTP interface
+# @param attractor.handlersNumber Total number of HTTP/GRPC handlers.
+# @param attractor.maxProcessTime Maximum processing time limit in minutes.
+# @param attractor.responseTimelimit Maximum response time limit in minutes.
+# @param attractor.requestTimeout Maximum request time limit in minutes.
+# @param attractor.dump.result Dump results in logs.
+# @param attractor.dump.query Dump queries in logs.
+# @param attractor.dump.answer Dump answers in logs.
+# @param attractor.logLevel Logging level, one of: Verbose, Info, Warning, Error, Fatal.
+# @param attractor.logMessageField Field name in logs for messages data.
+# @param attractor.grpcPort Container port for gRPC interface
+# @param attractor.indexFilename Name of the index file on Castle
+# @param attractor.citiesFilename Name of the cities file on Castle
+# @extra attractor.indices List of dynamic indices kill switches.
+# @skip attractor.indices.carOptions.enabled
+# @skip attractor.indices.countryCarOptions.enabled
+# @skip attractor.indices.ctxOptions.enabled
+# @skip attractor.indices.entrances.enabled
+# @skip attractor.indices.geoImport.enabled
+# @skip attractor.indices.passZones.enabled
+# @skip attractor.indices.projectCarOptions.enabled
+# @skip attractor.indices.snImport.enabled
+# @skip attractor.attractor
+# @param attractor.overrideConfig Complete config override. For test purposes only.
+# @param attractor.rtr.enabled Enable real time restrictions.
+# @param attractor.rtr.url URL real time restrictions server.
+# @param attractor.rtr.updatePeriod Update period from real time restrictions server.
+# @param attractor.mock If mock service being deployed. For test purposes only.
+# @param attractor.tilesMetricsThreshold The value at which we send tiles metrics (used for internal tests).
+# @skip rules
+
+attractor:
+  appPort: 8080
+  handlersNumber: 20
+  maxProcessTime: 20
+  responseTimelimit: 120
+  requestTimeout: 120
+  dump:
+    result: true
+    query: true
+    answer: false
+  logLevel: Info
+  logMessageField: custom.navi_msg
+  grpcPort: 50051
+  indexFilename: index.json.zip
+  citiesFilename: cities.conf.zip
+  indices:
+    # items in alphabet order
+    carOptions:
+      enabled: true
+    countryCarOptions:
+      enabled: true
+    ctxOptions:
+      enabled: true
+    entrances:
+      enabled: true
+    geoImport:
+      enabled: false
+    passZones:
+      enabled: true
+    projectCarOptions:
+      enabled: true
+    snImport:
+      enabled: false
+  attractor: {}
+  overrideConfig: ''
+  rtr:
+    enabled: false
+    url: http://rtr
```

нужно убрать дефолт и сделать обязательным параметром
```suggestion
    url: ''
```